RELEASING.md

```diff
@@ -1,30 +1,54 @@
-  - release `tkrelease -k="AWS_ACCESS_KEY" -s="AWS_SECRET_KEY" vX.X.X`
-    - If releasing a different version than in `src/release/version.go` you can use `-f` to force, sometimes this is necessary for specific issue tags like `v0.0.0-issue432` when trying to debug a issue.
-    - Using beta/alpha tags on the version number will stop themekit from automatically updating to that version. It would have to be typed in specifically like `theme update --version=v1.0.4-rc1`
-- On GitHub create a new release for the tag and take note of any relevant changes.
-  - Include a brief summary of all the changes
-  - Include links to the Pull Requests that introduced these changes
-- Update the [documentation website](https://shopify.github.io/themekit/)
-  - run `make serve_docs`
-  - update any changes to the API
-  - commit changes
-- (Consider waiting a day before performing the next steps, in case a defect is reported and we need to issue a patch release).
-- Update `themekitversion` in docs config `docs/_config.yml` to update the download links,
-  then run `make md5s` to generate the checksums for the new files. Add these to the `docs/_config.yml`
-  file as well.
-- Update Chocolatey package in the `choco` folder
-    - Update the version in `choco/themekit.nuspec`
-    - Update the version and checksums in `choco/tools/chocolateyinstall.ps1`
-    - Run `choco pack` in a window VM
-    - Log into [https://chocolatey.org/](chocolatey.org) (use the themekit@shopify.com credentials) and submit an update for approval
-- Update `themekit.rb` formula for homebrew on https://github.com/Shopify/homebrew-shopify
-  - run `make sha` to generate the SHA256 for the darwin build
-  - update the link, sha and version in the homebrew formula
+To release a new version of Theme Kit, make sure to complete all of the following steps.
+
+## 1. Update the Theme Kit version
+
+1. Update `ThemeKitVersion` in `src/release/release.go` and commit your changes.
+2. Make the release tool by running `go install ./cmd/tkrelease`.
+3. Run `git tag <version> && git push origin --tags && git push`.
+4. Update `changelog.txt` with the date of the version release.
+
+## 2. Release using tool
+
+1. Build all distributions by running `make all`.
+2. Release `tkrelease -k="AWS_ACCESS_KEY" -s="AWS_SECRET_KEY" vX.X.X`.
+
+    > Note:
+    > If you're releasing a different version than in `src/release/version.go`, then you can use `-f` to force. Sometimes this is necessary for specific issue tags like `v0.0.0-issue432` when trying to debug a issue.
+    > Using beta/alpha tags on the version number will stop `themekit` from automatically updating to that version. It would have to be typed in specifically (for example, `theme update --version=v1.0.4-rc1`).
+
+## 3. Create a new release on GitHub
+
+1. In GitHub, create a new release for the tag.
+2. Include a brief summary of all the changes pertaining to the release.
+3. Include links to the pull requests that introduced the changes.
```

Comment:
Because of my previous comment, we should add here to upload the binaries, and put the md5s in the description.